oshi-core/src/main/java/oshi/hardware/HWDiskStore.java

```diff
@@ -197,6 +201,13 @@ public long getWriteBytes() {
         return this.writeBytes;
     }
 
+    /**
+     * @return the length of the disk queue (#I/O's in progress)
```

Comment:
Given the multiple interpretations (e.g. average, current) this should probably be named `currentQueueLength`